webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -122,56 +123,64 @@ def _run_one(self, job_spec, force=False, cmd_args=None):
         start_time = None
         run_time = None
 
-        try:
-            cmd_kwargs = {}
-            with self.lock_job(job_spec['cmd']):
+        with self.lock_job(job_spec['cmd']):
```

Comment:
I moved the `lock_job` context outside all the exception handling. If exceptions happen, then the job will unlock.